[2022-07-28 10:47:43,623] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: user_processing.store_user scheduled__2022-07-28T09:00:00+00:00 [queued]>
[2022-07-28 10:47:43,631] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: user_processing.store_user scheduled__2022-07-28T09:00:00+00:00 [queued]>
[2022-07-28 10:47:43,631] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2022-07-28 10:47:43,632] {taskinstance.py:1357} INFO - Starting attempt 1 of 1
[2022-07-28 10:47:43,633] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2022-07-28 10:47:43,646] {taskinstance.py:1377} INFO - Executing <Task(PythonOperator): store_user> on 2022-07-28 09:00:00+00:00
[2022-07-28 10:47:43,651] {standard_task_runner.py:52} INFO - Started process 2557 to run task
[2022-07-28 10:47:43,654] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'user_processing', 'store_user', 'scheduled__2022-07-28T09:00:00+00:00', '--job-id', '45', '--raw', '--subdir', 'DAGS_FOLDER/user_processing.py', '--cfg-path', '/tmp/tmpl507bkce', '--error-file', '/tmp/tmp_7fz2cyz']
[2022-07-28 10:47:43,655] {standard_task_runner.py:80} INFO - Job 45: Subtask store_user
[2022-07-28 10:47:43,707] {task_command.py:369} INFO - Running <TaskInstance: user_processing.store_user scheduled__2022-07-28T09:00:00+00:00 [running]> on host 890323d6f4a0
[2022-07-28 10:47:43,776] {taskinstance.py:1571} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=user_processing
AIRFLOW_CTX_TASK_ID=store_user
AIRFLOW_CTX_EXECUTION_DATE=2022-07-28T09:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-07-28T09:00:00+00:00
[2022-07-28 10:47:43,777] {postgres.py:127} INFO - Running copy expert: COPY users FROM stdin WITH DELIMITER as ',', filename: /tmp/processed_user.csv
[2022-07-28 10:47:43,785] {base.py:68} INFO - Using connection ID 'postgres' for task execution.
[2022-07-28 10:47:43,791] {python.py:173} INFO - Done. Returned value was: None
[2022-07-28 10:47:43,803] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=user_processing, task_id=store_user, execution_date=20220728T090000, start_date=20220728T104743, end_date=20220728T104743
[2022-07-28 10:47:43,826] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-07-28 10:47:43,856] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
